src/NECompletion/NECModel.class.st

```diff
@@ -128,6 +128,16 @@ NECModel >> notEmpty [
 	^self isEmpty not
 ]
 
+{ #category : #printing }
+NECModel >> printOn: aStream [
+
+	super printOn: aStream.
+	clazz ifNotNil: 
```

Comment:
This is not related to this PR but this "clazz" seems weird. It should probably be "class". I tested a rename in a Pharo 7 image and it did not crash.